site/pages/[locale]/dp-auctions/index.js

```diff
@@ -0,0 +1,27 @@
+import { useRouter } from 'next/navigation'
+import { useEffect } from 'react'
+
+import { defaultLocale, locales } from '../../../navigation'
+
+function DpAuctionsIndex() {
+  const router = useRouter()
+
+  useEffect(
+    function () {
```

Comment:
Let's avoid using anonymous functions in `useEffect`, at least the in new ones. Adding a name to the function allow the reader understand what the effect is supposed to do.

Reply:
Renamed to `redirectToDefaultCollection`, and simplied a bit the implementation. here: https://github.com/hemilabs/pure.finance/pull/1/commits/abcda61f8357cd38f6d8a77b5ddadf6a64b163b9